### DISCUSSION TITLE
hermesengine.dev redirecting to GH ReadMe

### DISCUSSION THREAD

**Discussion by leotm** [facebook/hermes#1370] [Q&A] [answered]
curious what happened to the site? since couldn't find a github issue/discussion or tweet on it

wanted to check if the [playground](https://hermesengine.dev/playground/) ver is still latest release v0.12.0 (this [step](https://github.com/facebook/hermes/blob/main/.circleci/config.yml#L518-L540)), just shy of couple yrs old
if so can we default to latest tag instead (couple wks old) and when [building and running](https://hermesengine.dev/docs/building-and-running/) (GH [link](https://github.com/facebook/hermes/blob/main/doc/BuildingAndRunning.md)) Hermes

**Comment by tmikov** [comment, marked as answer]
We removed it, since it was pointless - just another rendering of the same documentation, already accessible from our Github project. It was maintenance headache, it was never up to date. Plus there was a continuous stream of security update issues (for a static website; give me a break...).

Hermes is not something for end users, our audience is engineers, so a GitHub project is perfect.

**Reply by leotm** [reply]
[5 year old](https://youtu.be/JsppO1HUYx4?si=Wx2vWV4F8gpMLQ9x) engineers too (;
